src/index.html

```diff
@@ -22,6 +22,388 @@
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-price">Price:</div>
+
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__iMac"
+          src="images/imac.jpeg"
+          alt="iMac 27"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-price">Price:</div>
+
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__iMac"
+          src="images/imac.jpeg"
+          alt="iMac 27"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-price">Price:</div>
+
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The product cards are repeated multiple times with the same content. Consider using a templating engine or JavaScript to dynamically generate these cards. This will make your code more maintainable and easier to update in the future.